Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-buildPlugin(useContainerAgent: true, configurations: [
+buildPlugin(useContainerAgent: false, configurations: [
```

Comment:
Temporary configuration in respect to https://github.com/jenkins-infra/helpdesk/issues/3105